src/tools/rls/LICENSE-MIT

```diff
@@ -0,0 +1,23 @@
+Permission is hereby granted, free of charge, to any
```

Comment:
Hm, are these files needed? Seems like the code would be licensed under the root LICENSE-* files anyway, which should be ~equivalent?

Reply:
Sure! I have changed it to use the root files instead.